docs/enroute/Melbourne Centre/ASP.md

```diff
@@ -28,6 +28,8 @@
 
 When **AS ADC** is offline, AS CTR (Class D and C `SFC` to `F125`) within 80 DME AS reverts to Class G, and AS CTR (Class C `F125` to `F245`) within 80 DME AS reverts to Class E, and both are administered by ASP. Alternatively, ASP may provide a [top-down procedural service](../../../aerodromes/Alice) if they wish.
 
+When **WR ADC** is offline, R222F (`SFC` to `F125`) reverts to Class G, and is administered by WRA. Alternatively, WRA may provide a [top-down procedural service](../../../aerodromes/Woomera) if they wish.
```

Comment:
This section can be removed, since the airspace is already Class G by default, and we don't want enroute controllers providing procedural top down. However, above the AS ADC airspace reclassification section, could you please add a:

### Reclassifications

header? Just like the format you'd see on other ENR/TCU pages like INL, for example